testbed/eks/test/testbed.test.ts

```diff
@@ -0,0 +1,13 @@
+import { expect as expectCDK, matchTemplate, MatchStyle } from '@aws-cdk/assert';
```

Comment:
I'd hope we don't have any logic in our stack, which means we shouldn't need to test anything.